include/sot/torque_control/free-flyer-locator.hh

```diff
@@ -105,21 +110,28 @@ namespace dynamicgraph {
         {
           getLogger().sendMsg("[FreeFlyerLocator-"+name+"] "+msg, t, file, line);
         }
-        
+
       protected:
+        
         bool              m_initSucceeded;    /// true if the entity has been successfully initialized
         se3::Model        m_model;            /// Pinocchio robot model
         se3::Data         *m_data;            /// Pinocchio robot data 
         se3::SE3          m_Mff;               /// SE3 Transform from center of feet to base
         se3::SE3          m_w_M_lf;
         se3::SE3          m_w_M_rf;
-        unsigned int      m_right_foot_id;
-        unsigned int      m_left_foot_id;
+        long unsigned int      m_right_foot_id;
+        long unsigned int      m_left_foot_id;
         Eigen::VectorXd   m_q_pin;            /// robot configuration according to pinocchio convention
         Eigen::VectorXd   m_q_sot;            /// robot configuration according to SoT convention
         Eigen::VectorXd   m_v_pin;            /// robot velocities according to pinocchio convention
         Eigen::VectorXd   m_v_sot;            /// robot velocities according to SoT convention
 
+	std::string m_Left_Foot_Frame_Name;
+	std::string m_Right_Foot_Frame_Name;
+	unsigned int m_nbJoints;
+	double m_Right_Foot_Sole_XYZ[3];
```

Comment:
I just realized that this is how the free-flyer-locator has been implemented, so ignore this comment!